<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMain</class>
 <widget class="QDialog" name="DialogMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WatchRR</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelStationInfo">
     <property name="text">
      <string notr="true">put info on station here</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabRR">
      <attribute name="title">
       <string>Precipitation Card</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="buttonEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Single values…</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonRedist">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Redistribute…</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonRedistQC2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Redistribute &amp;in QC2</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonAcceptRow">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Accept</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableRR">
         <property name="selectionMode">
          <enum>QAbstractItemView::ContiguousSelection</enum>
         </property>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>20</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabNeighborRR">
      <attribute name="title">
       <string>Neighbor Precipitation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="tableNeighborRR">
         <property name="selectionMode">
          <enum>QAbstractItemView::ContiguousSelection</enum>
         </property>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>20</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabNeighborData">
      <attribute name="title">
       <string>Neighbor Data</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="MiDateTimeEdit" name="dateNeighborData">
           <property name="displayFormat">
            <string notr="true">yyyy-MM-dd</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableNeighborData">
         <property name="selectionMode">
          <enum>QAbstractItemView::ContiguousSelection</enum>
         </property>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>20</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelInfo">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonUndo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>U&amp;ndo</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRedo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Redo is not yet implemented, sorry.</string>
       </property>
       <property name="text">
        <string>Redo</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Save</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MiDateTimeEdit</class>
   <extends>QDateEdit</extends>
   <header>MiDateTimeEdit.hh</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableRR</tabstop>
  <tabstop>buttonSave</tabstop>
  <tabstop>buttonClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonRedist</sender>
   <signal>clicked()</signal>
   <receiver>DialogMain</receiver>
   <slot>onRedistribute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonEdit</sender>
   <signal>clicked()</signal>
   <receiver>DialogMain</receiver>
   <slot>onEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSave</sender>
   <signal>clicked()</signal>
   <receiver>DialogMain</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1098</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogMain</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1189</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonUndo</sender>
   <signal>clicked()</signal>
   <receiver>DialogMain</receiver>
   <slot>onUndo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>870</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRedistQC2</sender>
   <signal>clicked()</signal>
   <receiver>DialogMain</receiver>
   <slot>onRedistributeQC2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>568</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAcceptRow</sender>
   <signal>clicked()</signal>
   <receiver>DialogMain</receiver>
   <slot>onAcceptRow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateNeighborData</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>DialogMain</receiver>
   <slot>onNeighborDataDateChanged(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onRedistribute()</slot>
  <slot>onEdit()</slot>
  <slot>onUndo()</slot>
  <slot>onRedistributeQC2()</slot>
  <slot>onAcceptRow()</slot>
  <slot>onShowNeighbors()</slot>
  <slot>onNeighborDataDateChanged(QDate)</slot>
 </slots>
</ui>
